test/interop/main_client.test.ts

```diff
@@ -0,0 +1,43 @@
+import { StreamClient} from "../../src";
+import { firstValueFrom, map, take, toArray } from "rxjs";
+
+const testEndpoint = "https://stream-api.cluster.amur-dev.proxima.one";
+
+describe("StreamClient", () => {
+  let client: StreamClient;
+  beforeEach(() => {
+    client = new StreamClient({});
+  });
+
+  it("should get stream endpoints and data", async () => {
+    const name = "eth-main-blockheader0.new-runtime"
```

Comment:
Let's not commit it with this stream name. It was temporary. The one you should use is "eth-main-blockheader".

Reply:
Doesn't `eth-main-blockheader` work right now? I believe it exists in kafka. Maybe we should update the relayer?